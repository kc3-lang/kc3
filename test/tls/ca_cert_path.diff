--- ca_cert_path.out.expected	Mon Sep  8 12:11:09 2025
+++ ca_cert_path.out	Mon Sep  8 12:11:18 2025
@@ -1,4 +1,27 @@
 require TLS
+cfn_link: kc3_tls_config_set_key_file: Unable to resolve symbol
+env_kc3_def: env_eval_tag
+cfn_apply: "kc3_def" [set_key = cfn Bool "kc3_tls_config_set_key_file" (TLS.Config*, Str)]: 0x0 != 0x749de4b9fbc0
+cfn_apply: "kc3_defmodule" [TLS.Config, do
+  dlopen("tls.so")
+  def delete = cfn Void "tls_config_free" (TLS.Config*)
+  def error = cfn Str "kc3_tls_config_error" (TLS.Config*, Result)
+  def new = cfn TLS.Config* "kc3_tls_config_new" (Result)
+  def set_ca_file = cfn Bool "kc3_tls_config_set_ca_file" (TLS.Config*, Str)
+  def set_cert_file = cfn Bool "tls_config_set_cert_file" (TLS.Config*, Str)
+  def set_key = cfn Bool "kc3_tls_config_set_key_file" (TLS.Config*, Str)
+end]: 0x0 != 0x749de4ba0370
+env_load: env_eval_tag: defmodule TLS.Config do
+  dlopen("tls.so")
+  def delete = cfn Void "tls_config_free" (TLS.Config*)
+  def error = cfn Str "kc3_tls_config_error" (TLS.Config*, Result)
+  def new = cfn TLS.Config* "kc3_tls_config_new" (Result)
+  def set_ca_file = cfn Bool "kc3_tls_config_set_ca_file" (TLS.Config*, Str)
+  def set_cert_file = cfn Bool "tls_config_set_cert_file" (TLS.Config*, Str)
+  def set_key = cfn Bool "kc3_tls_config_set_key_file" (TLS.Config*, Str)
+end
+env_module_load: TLS.Config: env_load
+env_module_ensure_loaded: module not found: TLS.Config
 true
 TLS.ca_cert_path()
-""
+"/etc/ssl/cert.pem"
