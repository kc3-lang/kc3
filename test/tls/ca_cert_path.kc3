quote require TLS
require TLS
quote TLS.ca_cert_path()
TLS.ca_cert_path()
