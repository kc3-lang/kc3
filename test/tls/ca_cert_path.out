require TLS
cfn_link: kc3_tls_config_set_key_file: Unable to resolve symbol
env_kc3_def: env_eval_tag
cfn_apply: "kc3_def" [set_key = cfn Bool "kc3_tls_config_set_key_file" (TLS.Config*, Str)]: 0x0 != 0x749de4b9fbc0
cfn_apply: "kc3_defmodule" [TLS.Config, do
  dlopen("tls.so")
  def delete = cfn Void "tls_config_free" (TLS.Config*)
  def error = cfn Str "kc3_tls_config_error" (TLS.Config*, Result)
  def new = cfn TLS.Config* "kc3_tls_config_new" (Result)
  def set_ca_file = cfn Bool "kc3_tls_config_set_ca_file" (TLS.Config*, Str)
  def set_cert_file = cfn Bool "tls_config_set_cert_file" (TLS.Config*, Str)
  def set_key = cfn Bool "kc3_tls_config_set_key_file" (TLS.Config*, Str)
end]: 0x0 != 0x749de4ba0370
env_load: env_eval_tag: defmodule TLS.Config do
  dlopen("tls.so")
  def delete = cfn Void "tls_config_free" (TLS.Config*)
  def error = cfn Str "kc3_tls_config_error" (TLS.Config*, Result)
  def new = cfn TLS.Config* "kc3_tls_config_new" (Result)
  def set_ca_file = cfn Bool "kc3_tls_config_set_ca_file" (TLS.Config*, Str)
  def set_cert_file = cfn Bool "tls_config_set_cert_file" (TLS.Config*, Str)
  def set_key = cfn Bool "kc3_tls_config_set_key_file" (TLS.Config*, Str)
end
env_module_load: TLS.Config: env_load
env_module_ensure_loaded: module not found: TLS.Config
true
TLS.ca_cert_path()
"/etc/ssl/cert.pem"
